spec/models/section_spec.rb

```diff
@@ -1,17 +1,33 @@
 require "rails_helper"
 
 RSpec.describe Section do
-  context 'when a section is updated' do
-    context 'when period information is updated' do
+    context 'when a section is updated' do
+      context 'notify Kafka topic' do
+         let(:param) { rand }
+         subject(:responder) { ::SectionsResponder.new }
+
+           describe '#send_notification' do
+                 it 'executes call method with self parameter' do
+                     expect(responder.call(self))
+                 end
+
+                 it 'cannot accept another parameter' do

```

Comment:
Not sure exactly why we need this, but it looks like this should go in the responder test class.